### DISCUSSION TITLE
Future Roadmap?

### DISCUSSION THREAD

**Discussion by haubourg** [CaussesCevennes/VOPP#5] [General]
Hi there, Salut @domlysz :)

I discover this nice and well documented tool that would partially fit a use case of my current employer, but in an urban environmement. 

Before testing anything, is there any roadmap for this tool already? 
What is the governance if we would want to push some enhancement or fixes? 

Last question, a better responsive design is probably the major missing feature for us, so is there anything already in the pipe? 

Thanks !

**Comment by domlysz** [comment]
Salut Régis, sympa de te croiser ici !

Je me permets de continuer en français.

Pas vraiment de roadmap pour le moment, le projet est en phase _snowball_. L'objectif pour moi est d'aider à conforter la base utilisateur pour mettre en place une forme de gouvernance via un comité d'utilisateurs. Quand des structures sont intéressées je peux proposer de l'accompagnement au déploiement ou à l'ajout de fonctionnalités via convention.

Pour le moment 2 parcs régionaux et un CAUE ont choisi cette solution. A l'avenir à minima un autre parc régional et un autre CAUE devraient nous rejoindre. J'aimerai surtout trouver des partenaires capables de mettre un peu de RH sur le volet développement afin de ne pas être seul décisionnaire sur les questions de design du code, en particulier parce que je maîtrise mal l'écosystème JS.

Les parcs Nationaux semblent s'engager vers une autre option développée dans leur réseau https://github.com/PnX-SI/GeoPaysages. Je suis également en lien avec la DREAL Occitanie qui anime un réseau paysage régional et qui souhaite s'engager dans la solution http://popp-breizh.fr pour centraliser les initiatives locales et proposer un outil facilitateur aux gestionnaires d'OPP.

Ces deux outils sont plus orientés backend que VOPP qui est avant tout centré sur la visualisation sans trop s'embarrasser du reste, ce qui je pense peut être vu comme une faiblesse ou comme un atout. Une possible synergie pourrait émerger avec le DREAL pour créer un pont entre nos travaux respectifs et éviter que les efforts des gestionnaires d'OPP ne se dispersent inutilement dans des solutions de valorisation opposées.

En terme de contribution, c'est donc pour le moment très libre ! En tout cas le projet intéresse ce qui me laisse espérer de belles perspectives d'évolution pour la suite.

Au plaisir de te lire

**Comment by haubourg** [comment]
Merci pour la réponse et pour le pointeur vers le projet GeoPaysages ! Je pense que ça pourra répondre à nos projets de plus long terme. A ce stade, on va faire une appli web minimale dans le framework en place, mais les fonctionnalités de votre outil vont certainement faire évoluer la commande. 
 
Je ferme le ticket ?
